sdk/openai/Azure.AI.OpenAI/tests/OpenAIInferenceTests.cs

```diff
@@ -100,11 +104,93 @@ public async Task EmbeddingTest()
         [RecordedTest]
         public void RequestFailedExceptionTest()
         {
-            var client = GetClient();
+            OpenAIClient client = GetClient();
             CompletionsOptions completionsRequest = new CompletionsOptions();
             completionsRequest.Prompt.Add("Hello world");
-            var exception = Assert.ThrowsAsync<RequestFailedException>(async () => { await client.GetCompletionsAsync("BAD_DEPLOYMENT_ID", completionsRequest); });
+            RequestFailedException exception = Assert.ThrowsAsync<RequestFailedException>(async () =>
+            {
+                await client.GetCompletionsAsync("BAD_DEPLOYMENT_ID", completionsRequest);
+            });
             Assert.AreEqual(404, exception.Status);
         }
+
+        [RecordedTest]
+        public async Task StreamingCompletionsTest()
```

Comment:
here.

Reply:
Yep, this is the place!
[posterity note: discussion about where the test is and code review bookmarking]